src/http/representation/ResourceIdentifier.ts

```diff
@@ -14,3 +14,10 @@ export interface ResourceIdentifier {
 export function isResourceIdentifier(object: any): object is ResourceIdentifier {
   return object && (typeof object.path === 'string');
 }
+
+/**
+ * Factory function creating a resource identifier for convenience.
```

Comment:
```suggestion
 * Factory function creating a {@link ResourceIdentifier} for convenience.
```
Not bad to have this indeed. Just did a quick search and apparently there are 100+ locations we would have to update to be consistent though so let's not look into that right now 😄